lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/update-name.yaml

```diff
@@ -209,9 +209,9 @@ states:
         targetState: FAILED_UPDATE_DETAILS
       alternate-doc-invalid-dl:
         targetJourney: FAILED
-        targetState: FAILED_UPDATE_DETAILS
+        targetState: FAILED_CONFIRM_DETAILS_INVALID_ID
```

Comment:
@Wynndow this is the crossover with DL auth source checks - we missed it because without the feature flag, `FAILED_UPDATE_DETAILS`/`FAILED_CONFIRM_DETAILS` both do the same thing!